torchok/data/datasets/classification/multihead.py

```diff
@@ -0,0 +1,134 @@
+from functools import partial
+from pathlib import Path
+from typing import Any, Optional, Union, Tuple, List, Dict
+
+import numpy as np
+import pandas as pd
+import torch
+from albumentations import BasicTransform
+from albumentations.core.composition import BaseCompose
+
+from torchok.constructor import DATASETS
+from torchok.data.datasets.base import ImageDataset
+from torchok.data.datasets.classification.classification import process_multilabel
+
+
+@DATASETS.register_class
+class MultiHeadImageDataset(ImageDataset):
+    TARGET_TYPES = ['multiclass', 'multilabel', 'feature_label']
+
+    def __init__(self,
+                 data_folder: str,
+                 annotation_path: str,
+                 targets: List[Dict[str, Any]],
+                 transform: Optional[Union[BasicTransform, BaseCompose]],
+                 augment: Optional[Union[BasicTransform, BaseCompose]] = None,
+                 input_column: str = 'image_path',
+                 input_dtype: str = 'float32',
+                 image_format: str = 'rgb',
+                 rgba_layout_color: Union[int, Tuple[int, int, int]] = 0,
+                 test_mode: bool = False,
+                 lazy_init: bool = False):
+        """Init ImageClassificationDataset.
+
+        Args:
+            data_folder: Directory with all the images.
+            annotation_path: Path to the .pkl or .csv file with path to images and annotations.
+                Path to images must be under column ``input_column`` and
+                annotations must be under ``target_column`` column.
+            targets: Dict containing information about heads
```

Comment:
`List of dicts where each dict contain information about heads` - I think this is better

Reply:
Added